<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwikeAnalyzer</class>
 <widget class="QMainWindow" name="TwikeAnalyzer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QTwikeAnalyzer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Decoder Log</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QTableView" name="tableLog">
          <property name="font">
           <font>
            <family>Arial</family>
            <weight>50</weight>
            <bold>false</bold>
           </font>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Battery 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <widget class="QwtPlot" name="qwtPlot1"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Battery 2</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_5">
        <item row="0" column="0">
         <widget class="QwtPlot" name="qwtPlot2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Battery 3</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QwtPlot" name="qwtPlot3"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <addaction name="actionFind"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuAnalyze">
    <property name="title">
     <string>&amp;Analyze</string>
    </property>
    <addaction name="actionRecord"/>
    <addaction name="actionStop"/>
    <addaction name="actionPause"/>
    <addaction name="actionPlay"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAnalyze"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionRecord"/>
   <addaction name="actionStop"/>
   <addaction name="actionPause"/>
   <addaction name="actionPlay"/>
   <addaction name="separator"/>
   <addaction name="actionDownload"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionRecord">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/media-record.png</normaloff>:/images/media-record.png</iconset>
   </property>
   <property name="text">
    <string>Record</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/media-playback-pause.png</normaloff>:/images/media-playback-pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/document-open.png</normaloff>:/images/document-open.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/media-playback-start.png</normaloff>:/images/media-playback-start.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/media-playback-stop.png</normaloff>:/images/media-playback-stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionDownload">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/mail-receive.png</normaloff>:/images/mail-receive.png</iconset>
   </property>
   <property name="text">
    <string>Download</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/document-save-as.png</normaloff>:/images/document-save-as.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save as</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="toolTip">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/file-find.png</normaloff>:/images/file-find.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Find</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/images/configure.png</normaloff>:/images/configure.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
